ContentPatcher/docs/i18n/zh/author-guide/tokens.md

```diff
@@ -0,0 +1,2011 @@
+         "Entries": {
+            "Mon": "Hard to imagine you only arrived {{query: {{DaysPlayed}} / 28}} months ago!"
+         },
+         "When": {
+            "query: {{Hearts:Abigail}} >= 10": true
+         }
+      }
+   ]
+}
+```
+
+文本值需用单引号包裹（包括返回文本的令牌）：
+```js
+"Query: '{{Season}}' = 'spring'": true
+```
+
+表达式不区分大小写，包括文本比较。
+
+### 注意事项
+查询表达式功能强大，但需注意：
+
+* 查询表达式 **很难验证** 无效表达式通常不会预先警告，仅在应用补丁时失败。需仔细测试表达式的功能，可以用[`patch parse`](troubleshooting.md#parse)检查新表达式。
+* 查询表达式 **可能错误读取文本** 例如，玩家名含单引号`D'Artagnan`，时，以下表达式会因语法错误失败：
+  ```js
+  "Query: '{{PlayerName}}' LIKE 'D*'": true // 'D'Artagnan' LIKE 'D*'
+  ```
+* 可能返回技术性错误信息。
+* 可能降低内容包可读性。
+
+尽可能使用非表达式功能。例如：
+
+<table>
+<tr>
+<th>使用查询表达式</th>
+<th>不使用查询表达式</th>
+</tr>
+<tr>
+<td>
+
+```js
+"When": {
+   "Query: {{Time}} >= 0600 AND {{Time}} <= 1200": true
+}
+```
+
+</td>
+<td>
+
+```js
+"When": {
+   "Time": "{{Range: 0600, 1200}}"
+}
+```
+
+</td>
+</tr>
+</table>
+
+### 运算符
+支持的运算符如下：
+
+* 算术运算（如`5 + 5`）：
+
+  运算符 | 效果
+  -------- | ---------
+  \+       | 加
+  \-       | 减
+  \*       | 乘
+  /        | 除
+  %        | 取模
+  ()       | 组合
+
+* 比较值（如`5 < 10`）：
+
+  运算符 | 效果
+  -------- | ---------
+  `<`      | 小于
+  `<=`     | 小于等于
+  `>`      | 大于
+  `>=`     | 大于等于
+  `=`      | 等于
+  `<>`     | 不等于
+
+* 逻辑运算符：
+
+  运算符 | 效果
+  -------- | ------
+  `AND`    | 与，两者都为真，如`{{Time}} >= 0600 AND {{Time}} <= 1200`。
+  `OR`     | 或，至少一者为真，如`{{Time}} <= 1200 OR {{Time}} >= 2400`。
+  `NOT`    | 非，取反，如`NOT {{Time}} > 1200`。
+
+* 使用`()`组合运算避免歧义：
+
+  ```js
+  "Query: ({{Time}} >= 0600 AND {{Time}} <= 1200) OR {{Time}} > 2400": true
+  ```
+
+* 检查值是否`IN`或`NOT IN`列表中：
+
+  ```js
+  "Query: '{{spouse}}' IN ('Abigail', 'Leah', 'Maru')": true
+  ```
+
+* 使用`LIKE`或`NOT LIKE`检查文本前缀/后缀。通配符`*`只能在字符串开始/结尾，且仅用于引号文本（例如 `LIKE '1'`会生效，但`LIKE 1`会报错）。
+
+  ```js
+  "Query: '{{spouse}}' LIKE 'Abig*'": true
+  ```
+
+<a name="mod-provided-tokens"></a>
+
+### 模组提供令牌 
+SMAPI模组可添加新令牌供内容包使用（见[_模组拓展性_](../extensibility.md)），用法与Content Patcher的令牌相同。例如，使用Json Assets的令牌：
+```js
+{
+   "Format": "2.7.0",
+   "Changes": [
+      {
+         "Action": "EditData",
+         "Target": "Data/NpcGiftTastes",
+         "Entries": {
+            "Universal_Love": "74 446 797 373 {{spacechase0.jsonAssets/ObjectId:item name}}",
+         }
+      }
+   ]
+}
+```
+
+使用模组提供的令牌需满足以下至少一项：
+* 提供令牌的模组是你的内容包的[必需依赖](https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest#Dependencies)。
+* 或使用令牌的补丁有不可变的`HasMod`条件列出该模组：
+  ```js
+  {
+     "Format": "2.7.0",
+     "Changes": [
+        {
+           "Action": "EditData",
+           "Target": "Data/NpcGiftTastes",
+           "Entries": {
+              "Universal_Love": "74 446 797 373 {{spacechase0.jsonAssets/ObjectId:item name}}",
+           },
+           "When": {
+              "HasMod": "spacechase0.jsonAssets"
+           }
+        }
+     ]
+  }
+  ```
+
+<a name="aliases"></a>
+
+### 别名 
+_别名_ 为现有令牌添加可选替代名称，仅影响内容包，可使用别名和原名。主要用于其他模组提供的长名令牌。
```

Comment:
> _别名_ 为现有令牌添加可选替代名称，仅影响内容包，可使用别名和原名。主要用于其他模组提供的长名令牌。

The original text says that aliases only apply to the _current_ content pack (i.e. another installed content pack won't see the alias); I'm not sure if that's clear in the translation?